src/jdbc-auth.md

```diff
@@ -1435,59 +1405,49 @@ descriptive name with the protocol to be used for the connection. It contains
 the following columns:
 
 `connection_id`
-: The unique integer associated with each connection. This value is 
-  generated automatically when a new entry is inserted into the 
-  `guacamole_connection` table.
+: A unique integer for each connection, generated automatically upon creating a
+  new entry in the `guacamole_connection` table.
 
 `connection_name`
-: The unique name associated with each connection. This value must be 
-  specified manually, and must be different from any existing connection 
-  name in the same connection group. References to connections in other 
-  tables use the value from `connection_id`, not `connection_name`.
+: A unique name for each connection. This value must be manually specified and
+  unique within the same connection group. Other tables reference connections
```

Comment:
What is this connection group the same as now? With these edits, this sentence doesn't make sense to me anymore.